code/web/src/modules/survey/SurveyPage.js

```diff
@@ -3,43 +3,203 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
+import { Helmet } from 'react-helmet'
 
 // UI Imports
-// import Card from '../../ui/card/Card'
-// import Button from '../../ui/button/Button'
-// import H4 from '../../ui/typography/H4'
-// import Icon from '../../ui/icon'
-// import { white, grey2, black } from '../../ui/common/colors'
+import { Grid, GridCell } from '../../ui/grid'
+import Card from '../../ui/card/Card'
+import { white, grey, grey2, black } from '../../ui/common/colors'
+import Button from '../../ui/button/Button'
+import { H3, H4 } from '../../ui/typography'
 
 // App Imports
 import { APP_URL } from '../../setup/config/env'
+import { routeImage, routes } from '../../setup/routes'
 import { messageShow, messageHide } from '../common/api/actions'
-import { remove, getListByUser } from '../subscription/api/actions'
+import { getSurveyProducts, parseSurveyItems } from './api/actions'
 
 // Component
 class SurveyPage extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {
+			isLoading: false,
+			parsedItems: {rocker: [], bohemian: [], business: [], artsy: []}
+    }
+	}
+	
+  static fetchData({ store }) {
+    return store.dispatch(getSurveyProducts())
+  }
 
-  // constructor(props) {
-  //   super(props)
-
-  //   this.state = {
-  //     isLoading: false
-  //   }
-  // }
+	componentDidMount() {
+		this.props.getSurveyProducts();
+	}
+	
+	componentDidUpdate() {
+		if (this.state.parsedItems.rocker.length === 0) {
+			const inSurveyItems = this.props.surveyProducts.products.filter(item => item.isSurvey);
+			const filteredItems = inSurveyItems.reduce((acc, item) => {
+				acc[item.style].push(item);
+				
+				return acc;
+			}, {rocker: [], bohemian: [], business: [], artsy: []}); 
+			this.setState({parsedItems: filteredItems})
+		}
+	}
```

Comment:
Bundling the survey items by style so the render() below can grab from each style accordingly